server/src/database/quieres/tasks/editTaskQuery.ts

```diff
@@ -0,0 +1,18 @@
+import connection from '../../config/connection';
+
+const editTaskQuery = (
+  id: number,
+  name: string,
+  description: string,
+  priority: string,
+  endDate: string,
+  status: string,
```

Comment:
I suggest to put all arguments in object, moreover, we can define interface for this object.